src/common/MultiSelect.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  Checkbox,
+  FormControl,
+  Input,
+  InputLabel,
+  MenuItem,
+  Select,
+  SelectChangeEvent,
+} from "@mui/material";
+import { styled } from "@mui/material/styles";
+import React, { useEffect, useState } from "react";
+
+const MultiSelectContainer = styled("div")({
+  margin: 10,
+  minWidth: 200,
+  maxWidth: 300,
+});
+
+interface MultiSelectProps {
+  options: string[];
+  selected: string[];
+  setOptions: (selections: string[]) => void;
+  name: string;
+}
+
+const MultiSelect: React.FC<MultiSelectProps> = ({
+  options,
+  selected,
+  setOptions,
+  name,
+}: MultiSelectProps) => {
+  const [selectedItems, setSelectedItems] = useState<string[]>([]);
```

Comment:
Can we try to do the same thing we did for `<SelectControl />` of  not having this component state, and only use parent state?